src/staging/src/templates/staticLanding1.js

```diff
@@ -101,6 +101,21 @@ const LandingPage = ({ pageContext }) => {
           </div>
         </div>
       )}
+      {/* Show breadcrumbs and title even if there's no HTMLArea */}
+      {linkContent.length === 0 && (
+        <>
+          <div className="d-none d-md-block static-content-container page-breadcrumbs">
+            <Breadcrumbs separator="›" aria-label="breadcrumb">
+              {renderBreadcrumbs(menuContent, pageContext?.page)}
+            </Breadcrumbs>
+          </div>
+          <div className="static-content--header">
+            <h1 className="header-title">
+              {page.Title}
+            </h1>
+          </div>
+        </>
+      )}
```

Comment:
Added breadcrumbs and title even if the page does not have `HTMLArea`

Reply:
Hmmm... This won't ever really be used, but it's a good reminder of how easy it will be to add the header via component. Nice.

Reply:
Yes, it's for the very edge case that the `staticLanding` page doesn't have any elements. This part will be changed once [this ticket](https://bcparksdigital.atlassian.net/browse/CM-143) is implemented 🙂